### PR TITLE
fixed dashboard

### PR DESCRIPTION
**Pull request by sami2ahmed**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: dashboards/confluent-cloud/confluent-cloud.json [hunk 1]
```diff
@@ -24,7 +25,7 @@
             },
             "nrqlQueries": [
               {
-                "accountId": 0,
+                "accountId": 3974333,
```
Comment (brammerl) [inline dashboards/confluent-cloud/confluent-cloud.json:28]: These should be 0

File: dashboards/confluent-cloud/confluent-cloud.json [hunk 2]
```diff
@@ -621,7 +622,7 @@
       "items": null,
       "defaultValues": [],
       "nrqlQuery": {
-        "accountIds": [],
+        "accountIds": [3974333],
```
Comment (brammerl) [inline dashboards/confluent-cloud/confluent-cloud.json:625]: This should be any empty array

**Review by brammerl — CHANGES_REQUESTED** [review]
Left some comments on how to improve this.

**Comment by sarahkitten** [comment]
@sami2ahmed these ids are intentional as this is a template to be used on any account - we're going to close this ticket now, feel free to reopen if you disagree

**Pull request closed without merging**